odoo/addons/base/models/res_country.py

```diff
@@ -71,6 +71,7 @@ class Country(models.Model):
         ], string="Customer Name Position", default="before",
         help="Determines where the customer/company name should be placed, i.e. after or before the address.")
     vat_label = fields.Char(string='Vat Label', translate=True, prefetch=True, help="Use this field if you want to change vat label.")
+    tax_label = fields.Char(string='Tax Label', help="Use this field if you want to change tax label for Point of Sale")
```

Comment:
Why not adding this field in POS if it is the only usage ?

Reply:
I see a similar task(3516548) in progress may be I can rework with the help description but it is a reusable and helpful

Reply:
@ryv-odoo The above mentioned task is now moved to this task, I will be doing the same for E commerce and Sale Mail Template, I wonder if we are using VAT in any-other module too (cc- @jco-odoo)

Now this PR is not only about POS